### PR TITLE
Refactor bitesize9

### PR DESCRIPTION
**Pull request by t-young31**

### CODE REVIEW

File: tdms/include/simulation_parameters.h [hunk 1]
```diff
@@ -8,19 +10,47 @@ struct PerfectlyMatchedLayer {
...
+
+enum RunMode{
+  complete,
+  analyse
+};
+
+enum Dimension{
+  THREE,  // Full dimensionality - compute all H and E components
+  TM,     // Transverse electric - only compute Ex, Ey, and Hz components
+  TE      // Transverse magnetic - only compute Hx, Hy, and Ez components
```
Comment (samcunliffe) [inline tdms/include/simulation_parameters.h]:
```suggestion
  TE,     // Transverse electric - only compute Ex, Ey, and Hz components
  TM      // Transverse magnetic - only compute Hx, Hy, and Ez components
```
Wrong way around?

File: tdms/src/iterator.cpp [hunk 2]
```diff
@@ -39,16 +40,6 @@ using namespace std;
...
 //parameter to control the with of the ramp when introducing the waveform in steady state mode
 #define ramp_width 4.
-//different run modes
-#define rm_complete    0
-#define rm_analyse     1
-//different source modes
-#define sm_steadystate 0
-#define sm_pulsed      1
-//'dimensionality' of simulation; controls which field modes to compute
-#define THREE 0    // Full dimensionality - compute all H and E components
```
Comment (samcunliffe) [inline tdms/src/iterator.cpp:49]: Bye bye 👋

**Review by samcunliffe — APPROVED** [review]
Sorry for being slow.
Looks good to me.
9️⃣ ✅

File: tdms/include/simulation_parameters.h (hunk 1 above)
Comment (t-young31) [reply tdms/include/simulation_parameters.h]: very much so 🤦🏼 – thanks for the spot

**Pull request merged**